config/routes.yaml

```diff
@@ -49,12 +49,18 @@ pulsearch:
     requirements:
        index_type: any|issn|isbn|title
 
-dpulsearch:
+pulfalightsearch:
     path: /dpulsearch/{index_type}
     defaults: { _controller: 'App\Controller\DpulSearchController::page' }
     requirements:
        index_type: any|issn|isbn|title
 
+dpulsearch:
```

Comment:
@sdellis I would have expected this one to be pulfalight and the previous one to be dpul